scripts/issue/9/claim_ibbtc_rewards.py

```diff
@@ -0,0 +1,21 @@
+from brownie import interface
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+def main():
+    ibbtc_msig = GreatApeSafe(registry.eth.badger_wallets.ibbtc_multisig)
+
+    bcvx = interface.ISettV4h(registry.eth.treasury_tokens.bCVX)
+    bvecvx = interface.ISettV4h(registry.eth.treasury_tokens.bveCVX)
+    bcvxcrv = interface.ISettV4h(registry.eth.treasury_tokens.bcvxCRV)
+
+    ibbtc_msig.take_snapshot(tokens=[
+        bcvx.address, bvecvx.address, bcvxcrv.address
+    ])
+
+    ibbtc_msig.init_badger()
+    ibbtc_msig.badger.claim_all_from_tree()
```

Comment:
There doesn't seem to be a `claim_all_from_tree()` method on Badger. Shouldn't it be `claim_all()`?

Reply:
good catch, i renamed that to match the signatures we use for the other classes, forgot to change it here.

Reply:
#d107eef